package.json

```diff
@@ -64,5 +65,10 @@
     "test": "npm run lint && npm run fast-test",
     "view-cover": "npm run cover -- --report=html && opn ./coverage/index.html"
   },
-  "private": false
+  "files": [
```

Comment:
You can use the `.npmignore` in place of this.

https://docs.npmjs.com/files/package.json#files